.github/workflows/issue-opened.yml

```diff
@@ -0,0 +1,68 @@
+name: Demo Workflow
+
+on:
+  issues:
+    types: [opened]
+
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  handle-issue:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Get Issue Creator
```

Comment:
do you want to trigger this workflow in all the issues? Optionally, we can use a "label" to filter which ones will be close with this

Reply:
it is triggered by default to keep it simple for the user, but we need to differentiate from development related issues; maybe by ignoring issues with a label?

otherwise we can setup an issue template with a specific "demo" label, I will investigate this second option

Reply:
While using the label seems to be the easiest solution, it does require manual assignment. A template, on the other hand, sounds like a great idea!

Reply:
this adjustment skip the demo workflow job for labeled issues (development issues): https://github.com/kudos-ink/demo/pull/7/commits/b0972cfca991efd2b9ce3f6fe2297944b7f15ac0